lib/registrant.rb

```diff
@@ -0,0 +1,31 @@
+class Registrant
+    attr_reader :age,
+                :license_data,
+                :name,
+                :permit
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @license_data = {
+            written: false,
+            license: false,
+            renewed: false
+            }
+        @permit = permit
+               
+    end
+
+    def permit?
+        @permit == @permit
+    end
+
+    def earn_permit
+        if @permit == false
+            @permit != false
+        end
+    end
```

Comment:
This method is not going to change the permit value. You'll need to reassign the value of `@permit` if the condition on line 24 evaluates to `true`. To do that, on line 25, you'll need it to say `@permit = true`